substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -633,4 +640,626 @@ mod e2e {
 			assert_ok!(Submissions::<Runtime>::ensure_killed(0));
 		})
 	}
+
+	#[test]
+	fn after_rejecting_calls_verifier_start_again_if_leader_exists() {
+		ExtBuilder::signed().build_and_execute(|| {
+			roll_to_signed_open();
+			assert_full_snapshot();
+
+			// Submit invalid solution with high score but no pages (will be slashed)
+			{
+				let mut strong_score = mine_full_solution().unwrap().score;
```

Comment:
not important here, but you don't need to actually mine anything to just obtain a fake large score. You can do `ElectionScore::max_value()`